pkg/preparer/orchestrate_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !race
```

Comment:
what race did you hit? looks like this file doesn't depend on consulutil which has known races, so if the race detector showed a problem it's probably a real problem